test/externalTests/brink.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2022 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -e
+
+source scripts/common.sh
+source test/externalTests/common.sh
+
+REPO_ROOT=$(realpath "$(dirname "$0")/../..")
+
+verify_input "$@"
+BINARY_TYPE="$1"
+BINARY_PATH="$2"
+SELECTED_PRESETS="$3"
+
+function compile_fn { yarn compile; }
+function test_fn { SNAPSHOT_UPDATE=1 npx --no hardhat test; }
+
+function brink_test
+{
+    local repo="https://github.com/brinktrade/brink-core"
+    local ref_type=branch
+    local ref=master
+    local config_file="hardhat.config.js"
+    local config_var=""
+    local extra_settings="metadata: {bytecodeHash: 'none'}"
+    local extra_optimizer_settings="runs: 800"
```

Comment:
What happens without this?

Reply:
It's needed for tests to pass-  they seem to depend on the exact bytecode on 0.8.10. With this tests on 0.8.10 pass.

The thing is, I can't get them to pass on any other versions so I gave up and they're disabled. So I could remove this since it's now compile only. But if we keep it, we won't have to rediscover that if we ever decide to get these tests to run.